newIDE/app/src/InstancesEditor/InstancesRenderer/LayerRenderer.js

```diff
@@ -478,12 +484,7 @@ export default class LayerRenderer {
     this._threeGroup = new THREE.Group();
     threeScene.add(this._threeGroup);
 
-    const threeCamera = new THREE.PerspectiveCamera(
-      this.layer.getCamera3DFieldOfView(),
-      1,
-      0.1,
-      2000
-    );
+    const threeCamera = new THREE.PerspectiveCamera(45, 1, 0.1, 2000);
```

Comment:
Why remove the camera fov of the layer?

Reply:
It can change at anytime, so it's set during the render (and not at the creation of the renderer) (both the layer configuration can change, and even the layer itself if swapped with another)

Reply:
Fine for me!